Source/Renderer/GLSLModernizer.js

```diff
@@ -0,0 +1,233 @@
+define([
```

Comment:
Not sure if that entails renaming the file to ModernizeShader, but I did that anyway